### DISCUSSION TITLE
Welcome to imls-pi-stack Discussions!

### DISCUSSION THREAD

**Discussion by jadudm** [18F/imls-pi-stack#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions for planning next steps in the minutes counting work:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MarisaPelczar** [comment]
Hello everyone!  I'm Marisa Pelczar, Program Analyst at the Institute of Museum and Library Services.  I worked with James and Matt J. (not to be confused with Matt B.) from GSA on designing this pilot.  I'm thrilled to join the discussion here (and learn about these Github discussion features too!).

**Comment by cantsin** [comment]
:wave: Hello all, I'm James, a software engineer over at [18F](https://18f.gsa.gov/) and I'm always happy to entertain ideas on how to improve anything here. Please, please feel free to ask questions as we work best with iteration and feedback. Lookin' forward to these discussions!

**Comment by VTLIB1** [comment]
Greetings, all! I'm Joshua Muse, the SDC for Vermont. I don't think I have a great deal of useful knowledge to offer here, but I am very enthusiastic about this project, and am happy to help if any opportunities come up. Thanks to everyone who is involved!
